test/server-unit/pdf.spec.js

```diff
@@ -36,17 +37,19 @@ function PDFRenderUnitTest() {
     expect(isBuffer && hasValidVersion).to.be.equal(true);
   });
 
-  it.skip('#pdf.render() renders as well #wkhtmltopdf the same PDF file from an HTML', async () => {
-    const result = await pdf.render({}, htmlFile, {});
-    const renderGenerated = result.toString();
+  it('#pdf.render() renders as well #wkhtmltopdf the same PDF file from an HTML', async () => {
 
-    const hasValidVersion = hasValidPdfVersion(renderGenerated);
-    const isBuffer = isBufferInstance(result);
-    expect(isBuffer && hasValidVersion).to.be.equal(true);
+    // load the HTML template into memory as a giant string
+    const tmpl = await fs.readFile(htmlFile, 'utf8');
+
+    // give the giant string to the render method
+    const rendered = await pdf.render({}, tmpl, {});
+    const cached = await fs.readFile(generatedFile);
 
-    // TODO: need a better to compare
-    // const wkGenerated = readFileToString(generatedFile);
-    // expect(renderGenerated).to.be.equal(wkGenerated);
+    expect(isBufferInstance(rendered)).to.equal(true);
+    expect(isBufferInstance(cached)).to.equal(true);
+
+    expect(rendered).to.deep.equal(cached);
```

Comment:
The difference is due to the creation date with a difference of 1 millisecond between rendered and cached.

You can see the difference if you convert them in string with .toString().

The challenge is how to ignore the `CreationDate` inside the PDF.

Reply:
Well, if it always is put in the same place, I suppose that we could just slice the buffer and ignore that portion.